src/app/dashboard/addproduct/page.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
```

Comment:
We have never accepted this endpoint (`src/app/dashboard/addproduct/page.tsx`). You need to revise our naming conventions.